game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/CombatValue.java

```diff
@@ -17,19 +18,21 @@
 
   boolean isDefending();
 
-  Collection<TerritoryEffect> getTerritoryEffects();
-
   GameData getGameData();
 
   Collection<Unit> getFriendUnits();
 
   Collection<Unit> getEnemyUnits();
 
+  CombatValue buildWithNoUnitSupports();
+
+  CombatValue buildOppositeCombatValue();
```

Comment:
The names for these two methods (`buildWithNoUnitSupports` and `buildOppositeCombatValue`) could probably be better.  And maybe there's a way to not need them but I couldn't figure one out.